sdk/nodejs/resource.ts

```diff
@@ -395,6 +397,25 @@ export interface ResourceOptions {
      * If this is a [ComponentResourceOptions] do not provide both [provider] and [providers]
      */
     provider?: ProviderResource;
+    /**
+     * An optional customTimeouts configuration block.
+     */
+    customTimeouts?: CustomTimeouts;
+}
+
+export interface CustomTimeouts {
+    /**
+     * The optional create timeout in minutes
```

Comment:
Wondering - Would we rather want this to be a count of seconds?

Reply:
I think for users to calculate seconds would be painful - the terraform model allows people to say 5m, 10m etc I have yet to see any work where seconds would suffice. We can revisit this of course

Reply:
I have explained the new rationale in the PR comments